docs/access_rights/organization_account.md

```diff
@@ -0,0 +1,60 @@
+> If you are interested in joining the closed beta testing, contact us at [support@apify.com](mailto:support@apify.com?subject=Organization%20account%20beta%20testing).
+
+Apify's organization account allows multiple team members to collaborate on projects. It enables you to manage your team members' [permissions]({{@link access_rights/list_of_permissions.md}}) and to centralize your billing.
+
+At the same time, a separate organization account means teams no longer have to share the credentials of a single personal account. You can [switch]({{@link access_rights/organization_account/how_to_use.md#in-the-apify-app}}) between your personal and organization accounts seamlessly in just two clicks.
+
+You can start an organization account in two ways.
+* [Convert an existing account](#convert-an-existing-account) into an organization. If your actors and integrations are set up in a personal account, it is probably best to convert that account into an organization. This will preserve all your integrations but means you will need a new personal account.
+* [Create a brand-new account](#create-a-new-organization). If you don't have integrations set up yet, or if they are easy to change, you can create a new organization, preserving your personal account.
+
+## [](#create-a-new-organization) Creating a new organization
+
+> **Warning**: the organization account is under closed beta testing. If you'd like to try it, contact us at [support@apify.com](mailto:support@apify.com?subject=Organization%20account%20beta%20testing).
+
+You can create a new organization by clicking the **Create a new organization** button under the **My organizations** tab in your [account](https://my.apify.com/account?redirect=/account#/myorganizations).
+
+![Create a new organization]({{@asset access_rights/images/my-organizations-new.png}})
+
+> You can own as many organizations and be a member of as many organizations as you need.
+
+An organization needs its own email account for receiving notifications, invoices, etc. If your email provider supports it, you can set up an [email alias](https://support.cloudhq.net/how-to-setup-gmail-aliases/), so the organization's emails get redirected to your email address.
+
+## [](#convert-an-existing-account) Convert an existing account
+
+> **Warning: when you convert an existing user account into an organization,**
+>  * **You will no longer be able to sign in to the converted user account.**
+>  * **An organization cannot be converted back to a personal account.**
+>  * **When converting or creating the account, you select an owner who will be responsible for [setting up]({{@link access_rights/organization_account/setup.md}}) the organization.**
+
+Before converting your current user account into an organization, create a new personal account for the organization's owner.
```

Comment:
Also mention that username is required to be able to convert user to an organization.